components/financial_assistance/db/seedfiles/translations/en/ic/financial_assistance.rb

```diff
@@ -173,7 +173,10 @@
   "en.faa.application_for_coverage" => "Application for Coverage",
   "en.faa.update_reminder" => "If you make changes to your %{year} coverage that should apply to your %{year2} coverage too, you will need to come back and update your %{year2} application. You can do this by copying your latest %{year} application and submitting it for %{year2}.",
   # Mec check
-  "en.faa.mc_success" => "It looks like you may already be enrolled in MaineCare or Cub Care. If you need to update information like your income, address, or who is in your household, contact the Office for Family Independence at <a href='tel:855-797-4357'>(855) 797-4357</a> to make these changes before completing a CoverME.gov application.",
+  "en.faa.mc_success_existing" => "It looks like you already have Medicaid coverage.",
+  "en.faa.mc_success_enroll" => "If you’re enrolled in Medicaid and need to renew your coverage or update your information, go to ",
+  "en.faa.mc_success_link" => "districtdirect.dc.gov",
+  "en.faa.mc_success_phone" => " or contact District Direct at (202) 727-5355 / TTD: (800) 877-8339.",
```

Comment:
Utilize RR medicaid_agency_phone setting here.
Additionally, introduce a new RR setting for TTD number.